### DISCUSSION TITLE
Supported hardware

### DISCUSSION THREAD

**Discussion by lagnat** [mriscoc/Ender3V2S1#857] [General]
I've seen a lot of discussion around various boards with different SOCs.  Some work better than others, etc..  Same with screens.  It would be really useful if there was a list (maybe similar to this http://wiki.fluidnc.com/en/hardware/existing_hardware) with the hardware specifics, what works, what doesn't, how to identify, pros, cons, etc..  

My reason for asking is that I'm researching a 3D printer retrofit and I'd like to know what the best options are for boards and screens.

**Comment by mriscoc** [comment]
Good Idea, you can put the list here using as starting point the requirements given here: https://github.com/mriscoc/Ender3V2S1/wiki#requirements and the list support in the Configurator: 
![](https://raw.githubusercontent.com/mriscoc/Special_Configurations/main/images/Configurator.png)
